### ISSUE TITLE
Outage: failing to provision job nodes

### ISSUE THREAD

**Issue by rarkins** [Shippable/support#2906]
Same as #2873 two weeks ago - provisioning job nodes is timing out. Example: https://app.shippable.com/runs/57bec0f36467880f00d85220/1/console

**Comment by rarkins** [comment]
@kanthchandra please take a look. AWS is not reporting any errors:

![image](https://cloud.githubusercontent.com/assets/6311784/17966022/c6269264-6abf-11e6-8f1a-7e0477d3d32b.png)

**Comment by rarkins** [comment]
All these timing out:

![image](https://cloud.githubusercontent.com/assets/6311784/17966296/148f4864-6ac1-11e6-94df-48755becebd4.png)

**Comment by rarkins** [comment]
A build worked again. Outage time was approximately one hour. As this now seems to be a recurring issue, can you provide a root cause explanation?

**Comment by kanthchandra** [comment]
@rarkins sorry that I couldn't reply to you earlier.

We did a scheduled maintenence today and when doing so we missed to update one of the images to the latest version on the unstable ami. So whenever the machine booted it was pulling the latest image which took a little time and as a result the container wouldn't come up sometimes. I have now updated the ami with the latest version of the image. So everything should be back to normal.
